### ISSUE TITLE
Execute custom query action

### ISSUE THREAD

**Issue by ElBabbler** [elasticio/jdbc-component#26]
Add component action which allows to execute custom query in selected DB

**Comment by A3a3e1** [comment]
The estimation is quite big. It includes development for 4 DB types, full QA cycle and creation integration tests for them to make QA life easier.

**Comment by ElBabbler** [comment]
- [x] All the branches if the repo should be checked (merged/deleted if possible) before starting development
- [x] Development team has all needed requirements
- [x] Access to the needed service with all needed permissions is granted
- [x] No open tasks that could block Component development process
- [x] The E.io’s GitHub repository must be checked if there is another implementation of the component under development. If it exists, new repository should not be created
- [x] Component architecture should be discussed and approved before start coding

**Development:**

- [x] Code is written and formatted in accordance with the Styling guide
- [x] Code (incl. configs, test samples, environment variables etc.) MUST NOT contain any sensitive data like passwords, tokens, API keys etc.
- [x] Component logs does not contain any sensitive data (credentials, input/output requests, Environment variables)
- [x] Code is covered with unit tests
- [x] Component code is covered with integration tests (optional)
- [ ] Documentation (README.md file in the Github) is 100% ready
- [x] Changelogs (CHANGELOG.md) are created/updated
- [x] package.json or build.gradle is updated:
  - Component's version is updated according to Semantic Versioning 2.0.0
  - Component should be dockerised, add: "buildType":"docker"
- [x] CI is set up
- [ ] All development branches are reviewed

**After QA:**

- [ ] All development branches are merged into master branch
- [ ] Sprint review preparation is finished (demo flow is built)

**QA:**

- [ ] Test cases for the component are ready at the TestRail
- [ ] Functional testing is finished
- [ ] Regression testing is finished (optional)
- [ ] Component is deployed to the production stage (app.elastic.io) and/or other required stage
  - Component has a proper access level
  - Required Environment Variables are set
  - Smoke test is finished
- [ ] Release notes is done
- [ ] Component Roadmap is updated
- [ ] Component spreadsheet is updated
